doc/INSTALL.md

```diff
@@ -258,31 +257,31 @@ This is an example how to use the modular building environment of PIConGPU.
       command will be shown in the first line of the output.
     - `case001` is the directory were the full compiled binary with all
       parameter files will be installed to
-6. **Compile:** `make [-j]`: compile PIConGPU with the current parameter set: `case001`
-7. **Install:** `make install`: copy binaries and params to a fixed data structure: `case001`
-8. `cd ~/paramSets/case001`: goto installed programm
-9. **Run:** Example run for the HPC System "hypnos" using a PBS batch system
+7. **Compile:** `make [-j]`: compile PIConGPU with the current parameter set: `case001`
+8. **Install:** `make install`: copy binaries and params to a fixed data structure: `case001`
+9. `cd $PICHOME/paramSets/case001`: goto installed programm
+10. **Run:** Example run for the HPC System "hypnos" using a PBS batch system
     - *optional:* `tbg --help`
     - `tbg -s qsub -c submit/0016gpus.cfg
-           -t submit/hypnos/k20_profile.tpl ~/runs/testBatch01`
```

Comment:
How about adding a line here about how to start a job with bash (since bash is in the most cases available)
- ` tbg -s "bash" -c submit/0001gpus.cfg -t submit/bash/bash_mpiexec.tpl  $PICHOME/runs/testBatch01`